ksp_plugin_adapter/flight_planner.cs

```diff
@@ -817,6 +842,9 @@ class FlightPlanner : VesselSupervisedWindowRenderer {
   private const double log10_time_upper_rate = 7.0;
 
   private const int max_flight_plans = 10;
+
+  private double optimization_altitude = 10e3;
```

Comment:
Should this be written `1e4`?

Reply:
I tend to count in kilometres, not myriametres.